service/lib/dinstaller/dbus/service.rb

```diff
@@ -103,12 +103,6 @@ def storage_proposal_dbus
       def storage_actions_dbus
         @storage_actions_dbus ||= DInstaller::DBus::Storage::Actions.new(@logger)
       end
-
-      def installer
-        # TODO: this god object should not be needed anymore when all dbus API is adapted
-        # just that probe should  be kept to get installer probed ASAP
-        @installer ||= DInstaller::Manager.instance.tap(&:probe)
```

Comment:
yeap, I overlook method when I do merge of upstream where imo remove its usage from language. And it is also reason why probe is not called. We found it during coding session for adapting software in frontend and it is part of https://github.com/yast/d-installer/pull/49